x/authority/keeper/authorization_list.go

```diff
@@ -0,0 +1,25 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/zetacore/x/authority/types"
+)
+
+// SetAuthorizationList sets the authorization list to the store.It returns an error if the list is invalid.
+func (k Keeper) SetAuthorizationList(ctx sdk.Context, list types.AuthorizationList) {
+	store := ctx.KVStore(k.storeKey)
```

Comment:
We don't use a prefix, meaning it is written at the root of the module store

https://github.com/zeta-chain/node/blob/6423a338eb9a745315e477c0048c95eea5923593/x/authority/keeper/policies.go#L12